src/main/java/org/broadinstitute/hellbender/engine/FeatureInput.java

```diff
@@ -229,32 +251,14 @@ public String getName() {
      *
      * @return file backing this source of Features
      */
-    public File getFeatureFile() {
+    public String getFeaturePath() {
         return featureFile;
     }
 
     /**
-     * Gets the type of Feature contained in our file
-     *
-     * @return the type of Feature contained in our file
-     */
-    public Class<? extends Feature> getFeatureType() {
-        return featureType;
-    }
-
-    /**
-     * Sets the type of Feature contained in our file. Called by the engine after construction time.
-     *
-     * @param featureType the type of Feature contained in our file
-     */
-    protected void setFeatureType(final Class<? extends Feature> featureType) {
-        this.featureType = featureType;
```

Comment:
There is still a `featureType` field in this class -- I think you should restore these.